### DISCUSSION TITLE
Depends on which conditions, rancher desktop marks specific kubernetes version as "stable"

### DISCUSSION THREAD

**Discussion by jwenjian** [rancher-sandbox/rancher-desktop#4057] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/25657798/221490490-80c62230-9425-47fb-8f84-e134175e62e3.png)

In above screenshot, there is 2 labels for v1.25.4

- stable
- latest

I assume that the `latest` label is because in that moment, the v1.25.4 is the latest release of v1.25.

What is the reason to assign `stable` label to a specific version?

**Comment by jandubois** [comment, marked as answer]
These labels come from the k3s update channel. See https://docs.k3s.io/upgrades/manual for the definition of "latest" vs. "stable".

**Reply by jwenjian** [reply]
Got it, Thanks~
